### ISSUE TITLE
Build fails in Unity 2018.1.0b9

### ISSUE THREAD

**Issue by dmonego** [googlevr/gvr-unity-sdk#861]
Summary:
When trying to build in the latest beta version of unity, the build will fail with the following message:

`\Assets/GoogleVR/Editor/GvrBuildProcessor.cs(29,7): error CS0535: 'GvrBuildProcessor' does not implement interface member 'UnityEditor.Build.IPreprocessBuild.OnPreprocessBuild(UnityEditor.Build.Reporting.BuildReport)'
`

Found using:
 * Google VR SDK version: 1.120.0
 * Unity version: 2018.1.0b9
 * Phone manufacturer, model, and O/S version: any iOS
 * Viewer manufacturer & model: N/A

Steps to reproduce the issue:
 1. Start unity
 1. Import HelloVr scene.
 1. Configure as per ios quickstart documentation
 1. Build

Workarounds:
Don't use a beta release.

Additional comments:

**Comment by rusmaxham** [comment]
This will be fixed in 1.130.

**Comment by rusmaxham** [comment]
In the meantime, you can work around the issue by adding this to the body of the GvrBuildProcessor class:

```C#
#if UNITY_2018_1_OR_NEWER
  public void OnPreprocessBuild(BuildReport report) {
    OnPreprocessBuild(report.summary.platform, report.summary.outputPath);
  }
#endif
```

**Comment by lopis** [comment]
@rusmaxham When adding that snippet, I'm getting

    Assets/GoogleVR/Editor/GvrBuildProcessor.cs(46,34): error CS0246: The type or namespace name `BuildReport' could not be found. Are you missing `UnityEditor.Build.Reporting' using directive?


...even though I'm already using `UnityEditor.Build.Reporting`.

**Comment by rusmaxham** [comment]
Fixed in [v1.130](https://github.com/googlevr/gvr-unity-sdk/releases).
